clang/test/CXX/drs/cwg4xx.cpp

```diff
@@ -36,6 +36,13 @@ namespace cwg400 { // cwg400: 2.7
   // expected-error@-1 {{member 'a' found in multiple base classes of different types}}
   //   expected-note@#cwg400-A {{member type 'cwg400::A::a' found by ambiguous name lookup}}
   //   expected-note@#cwg400-B {{member type 'cwg400::B::a' found by ambiguous name lookup}}
+  struct F : A {};
+  struct G : A {
+    using G::A;
```

Comment:
```suggestion
    using G::a;
```
Did you mean lowercase `a`?

Reply:
> Did you mean lowercase `a`?

No. This is the form `using CurrentClass::BaseClass;`, which is incorrectly accepted before this PR in C++98 mode.

Reply:
Inheriting constructors are sufficiently special that I think we should test both `G::A` and `G::a`.